src/main/java/com/example/CST4383Frontend/controller/CityController.java

```diff
@@ -25,6 +28,22 @@ public String getCityInfo(@PathVariable("city") String cityName,
         }
 
         model.addAttribute("cityInfo",cityInfo);
+        model.addAttribute("levels",levels);
```

Comment:
I thought that this was not necessary, but I found that you provided levels in java code and list them in html code. Good job.